src/components/quiz-list/QuizListContainer.js

```diff
@@ -82,25 +83,31 @@ class QuizListContainer extends React.Component {
     render() {
         const quizes = this.state.upcomingQuizes;
         return (
-            <div>
+                <div>
                 {
                     /* Component with a button to start a demo quiz */
                     this.state.showHostYourOwnQuiz &&
                     <DemoQuiz
                         startDemoQuiz={this.handleClick}
                     />
                 }
-
-                {
-                    /* Components for upcoming quiz with a button to enter */
-                    quizes && quizes
-                        .map(quiz => <QuizList key={quiz.id}
-                            quizName={quiz.quiz_name}
-                            startTime={quiz.Start_time}
-                            enterQuiz={(event) => { this.handleStartQuiz(event, quiz.id) }}
-                            remainingTime={quiz.Start_time - Date.now()} />)
-                }
-            </div>
+                    <QuizContext.Consumer>
+                    { (context) => (
+                        <div>
+                        {
+                        /* Components for upcoming quiz with a button to enter */
+                        quizes && quizes
```

Comment:
@js-sapphire there's a typo. quizes should be quizzes. Find and replace this everywhere.